src/partials/MusicSheet.js

```diff
@@ -17,18 +17,36 @@ const MusicSheetRouter = ({
 
   const indent = isExercise ? null : <p style={{ textIndent: 50 }} />
 
+  const defaultEngraverParams = {
```

Comment:
Tämän voisi heittää vielä ylemmälle tasolle (Samalle tasolle kun `const NotesAndSoundMusicSheet` jne) niin sitten sitä ei tarvisi heitellä alaspäin propseilla

Reply:
ok